code/controllers/subsystems/trade.dm

```diff
@@ -137,7 +137,7 @@ SUBSYSTEM_DEF(trade)
 	return GLOB.price_cache[path]
 
 /datum/controller/subsystem/trade/proc/get_export_cost(atom/movable/target)
-	. = round(get_cost(target) * 0.6)
+	. = round(get_cost(target) * 1)
```

Comment:
```suggestion
	. = round(get_cost(target))
```
Bruh.

Reply:
I dont actually know how to code

Reply:
Also what I did before was tested. If this ends up like the platinum where it was suggested but didnt actually work and runtimed it's not on me

Reply:
It's possible to, you know, test it again. Also there is no suggestions on solidification PR, what you're talking about?

Reply:
I'll see if I remember to change and test it with the pipe stuff I'm doing